nexus/Cargo.toml

```diff
@@ -47,7 +48,7 @@ oximeter-db = { path = "../oximeter/db/" }
 parse-display = "0.6.0"
 # See omicron-rpaths for more about the "pq-sys" dependency.
 pq-sys = "*"
-propolis-client = { git = "https://github.com/oxidecomputer/propolis", rev = "e58937fea67aca6bc2d37a3cc50a72291328ae0e", features = [ "generated" ] }
```

Comment:
Note propolis#241 needs to be merged, then the propolis rev in this PR plus the relevant package-manifest bits need to be updated, before this PR goes in!

Reply:
https://github.com/oxidecomputer/propolis/pull/241